### DISCUSSION TITLE
GeoMAP | French SRID Lambert93 <--> US SRID WGS84, Convert, or... ?)

### DISCUSSION THREAD

**Discussion by SlodeSoft** [grafana/grafana#42652] [Plugins]
Hi,

We've got some data stored in projection mode - Official French "Lambert 93"; called SCR in french | SRID : System Reference Identifier.
GeoMap in actual state doesn't accept Lambert93, Is it possible to get a conversion or accept this data type coordonates, else how to convert Data in WGS84 format (standard US Long Lag)

Thanks in advance !

**Comment by SlodeSoft** [comment]
First solution : https://gis.stackexchange.com/questions/78838/converting-projected-coordinates-to-lat-lon-using-python

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
